ci/gpu/build.sh

```diff
@@ -108,6 +110,11 @@ python setup.py install
 
 cd $WORKSPACE
 
+################################################################################
+# CHECK - Run clang-tidy
+################################################################################
+
+python ./cpp/scripts/run-clang-tidy.py -cdb ./cpp/build/compile_commands.json
```

Comment:
I believe this should be in `ci/checks/style.sh` instead. There appears to be some clang stuff there already https://github.com/rapidsai/cuml/blob/branch-0.14/ci/checks/style.sh#L70

Again with the image changes in progress the versions of clang you have above will be available at all stages